qa/rpc-tests/sc_cert_nonceasing.py

```diff
@@ -118,32 +118,48 @@ def run_test(self):
         mark_logs("Node0 confirms sc creation generating 2 blocks", self.nodes, DEBUG_MODE)
         self.nodes[0].generate(3)
 
+        # #------------------------------------------------
```

Comment:
Are those commented lines (121 to 128) just old copy-pasted code that can be removed?

Reply:
It tested a condition that was supposed to fail in the past version and has been superseded by test in the lines immediately below them.
Removed in dfeb52fab30e3b941b44631be6cdf8a95f317a67